MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm

```diff
@@ -887,6 +953,33 @@ static MVKBarrierStage commandUseToBarrierStage(MVKCommandUse use) {
 	return mvkMTLScissorRectFromVkRect2D(clipToRenderArea(mvkVkRect2DFromMTLScissorRect(scissor)));
 }
 
+// Attachment locations are changing. Neeed to store attachments and beginning another Metal renderpass.
+void MVKCommandEncoder::updateColorAttachmentLocations(const MVKArrayRef<uint32_t> colorAttLocs) {
+	if (_mtlRenderEncoder && isDynamicRendering()) {
+		auto* mvkSP = getSubpass();
```

Comment:
Could this ever be `nullptr` at this point? The change you made above implies it could be...